docs/conf.py

```diff
@@ -19,10 +19,7 @@ def setup(app):
     )
     app.add_transform(AutoStructify)
     app.add_stylesheet("custom.css")
-    app.add_stylesheet("link_gen/vendor/bootstrap-4.1.3/css/bootstrap.min.css")
-    app.add_javascript("link_gen/vendor/jquery-3.3.1.slim.min.js")
     app.add_javascript("link_gen/link.js")
-    app.add_javascript("link_gen/vendor/bootstrap-4.1.3/js/bootstrap.bundle.min.js")
```

Comment:
Is it safe to delete these files from the `link_gen/vendor/` folder? https://github.com/jupyterhub/nbgitpuller/tree/master/docs/_static/link_gen/vendor